api/src/config/website.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Exports client website metadata-related enums.
+ */
+
+const layoutOptions = ['1', '2', '3'];
```

Comment:
nit: could we think of giving a name for our layouts other than 1,2,3?

Reply:
can't think of anything good right now, do you have any suggestions?